exercises/beer-song/description.md

```diff
@@ -1,4 +1,4 @@
-Produce the lyrics to that beloved classic, that field-trip favorite: 99 Bottles of Beer on the Wall.
+Recite the verses to that beloved classic, that field-trip favorite: 99 Bottles of Beer on the Wall.
```

Comment:
The change to `Recite` is good.

"Recite the lyrics" is better (and more consistent) than "Recite the verses"

Reply:
Done

Reply:
thanks.